### DISCUSSION TITLE
How do I calculate coverage for my unit tested vector configurations?

### DISCUSSION THREAD

**Discussion by nitrocode** [vectordotdev/vector#21552] [General]
https://vector.dev/docs/reference/configuration/unit-tests/

Is it possible to calculate coverage ? If not, is this a planned feature? 

We're hoping we can set thresholds of coverage such as 10% or 20% so when new vector configs are added, if the coverage falls below the threshold, then the PR would be unable to be merged until the coverage is above the minimum again.

**Comment by jszwedko** [comment]
That would be cool! Not a planned feature though. This is the first I've heard the request. Would you want to open this as a feature request so that others can indicate if they would find it valuable?

**Reply by nitrocode** [reply]
Thanks. I opened this feature request https://github.com/vectordotdev/vector/issues/21555
